### ISSUE TITLE
Set "releases_created" (and several other outputs) are even when release doesn't happen

### ISSUE THREAD

**Issue by kurtisvg** [google-github-actions/release-please-action#688]
### TL;DR

"releases_created" says "true if the release was created, false otherwise", but is unset if a release wasn't created. This is true for most of the outputs described on this page: https://github.com/google-github-actions/release-please-action#outputs.

### Expected behavior

I expected the output to be set to false as described in the documentation. 

### Observed behavior

The outputs were all unset. 

### Action YAML

```yaml
N/A
```

### Additional information

Having the output unset is really hard to debug -- it would be better if they are explicitly set (even to "" or "none" for the ones that aren't used when a release isn't made) so that these values can still be used.

**Comment by aasseman** [comment]
It's also a problem for me. I was trying to rely on the output values of major/minor/patch to get the current version, even when no new release is created.

**Comment by chingor13** [comment]
This is fixed in v4. We set the `releases_created` and `prs_created` if we tried to create them.

If you set, `skip-github-release: true`, then the `releases_created` output is omitted. Similarly, if you set `skip-github-pull-request: true`, then the `prs_created` output is omitted.
